### PR TITLE
MFA page grammatical fixes

### PR DESCRIPTION
**Pull request by 0rdinant**
I was sleepy when I made these edits so please make sure that I didn't obliterate some obscure portion of the page

### CODE REVIEW

File: docs/security/multi-factor-authentication.en.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 title: "Multi-factor Authentication"
 icon: 'material/two-factor-authentication'
 ---
-**Multi-factor authentication** (MFA, or 2FA) is a security mechanism that requires additional steps beyond entering your username (or email) and password. The most common method are time limited codes you might receive from an SMS or app.
+**Multi-factor authentication** (MFA, or 2FA) is a security mechanism that requires additional steps beyond entering your username (or email) and password. The most common methods are time limited codes you might receive from an SMS or app.
```
Comment (jonaharagon) [inline docs/security/multi-factor-authentication.en.md:5]:
```suggestion
**Multi-factor authentication** (MFA, or 2FA) is a security mechanism that requires additional steps beyond entering your username (or email) and password. The most common method is time limited codes you might receive from an SMS or app.
```

File: docs/security/multi-factor-authentication.en.md [hunk 2]
```diff
@@ -2,9 +2,9 @@
...
+The idea behind MFA is that even if a hacker (or adversary) is able to figure out your password (something you *know*), they will still need a device you own, like your phone, (something you *have*) in order to generate the code needed to log in to your account. MFA methods vary in security based on this premise: The more difficult it is for an attacker to gain access to your MFA method, the better. Examples of MFA methods (from weakest to strongest) include [SMS codes](https://en.wikipedia.org/wiki/One-time_password#SMS), email codes, app push notifications, [Time-based One-time Passwords (TOTP)](https://en.wikipedia.org/wiki/Time-based_one-time_password), Yubico OTP, and [FIDO2](https://en.wikipedia.org/wiki/FIDO2_Project) / [U2F](https://en.wikipedia.org/wiki/Universal_2nd_Factor).
```
Comment (jonaharagon) [inline docs/security/multi-factor-authentication.en.md:7]: I'm not sure that this change makes sense.

```suggestion
The idea behind MFA is that even if a hacker (or adversary) is able to figure out your password (something you *know*), they will still need a device you own like your phone (something you *have*) in order to generate the code needed to log in to your account. MFA methods vary in security based on this premise: The more difficult it is for an attacker to gain access to your MFA method, the better. Examples of MFA methods (from weakest to strongest) include [SMS codes](https://en.wikipedia.org/wiki/One-time_password#SMS), email codes, app push notifications, [Time-based One-time Passwords (TOTP)](https://en.wikipedia.org/wiki/Time-based_one-time_password), Yubico OTP, and [FIDO2](https://en.wikipedia.org/wiki/FIDO2_Project) / [U2F](https://en.wikipedia.org/wiki/Universal_2nd_Factor).
```

File: docs/security/multi-factor-authentication.en.md [hunk 3]
```diff
@@ -14,55 +14,55 @@ Receiving codes either from **SMS** or **email** are one of the weaker ways to s
...
 
 ### Time-based One-time Password (TOTP)
 
-**TOTP** is one of the most commons form of MFA available. When a user sets up TOTP they are generally required to scan a [QR Code](https://en.wikipedia.org/wiki/QR_code) which establishes a "shared secret" with the service that they intend to use. The shared secret is secured inside of the authenticator app's data, and is sometimes protected by a password.
+**TOTP** is one of the most commons form of MFA available. When a user sets up TOTP they are generally required to scan a [QR Code](https://en.wikipedia.org/wiki/QR_code) which establishes a ["shared secret"](https://en.wikipedia.org/wiki/Shared_secret) with the service that they intend to use. The shared secret is secured inside of the authenticator app's data, and is sometimes protected by a password.
```
Comment (jonaharagon) [inline docs/security/multi-factor-authentication.en.md:25]:
```suggestion
**TOTP** is one of the most commons form of MFA available. When a user sets up TOTP they are generally required to scan a [QR Code](https://en.wikipedia.org/wiki/QR_code) which establishes a "[shared secret](https://en.wikipedia.org/wiki/Shared_secret)" with the service that they intend to use. The shared secret is secured inside of the authenticator app's data, and is sometimes protected by a password.
```

File: docs/security/multi-factor-authentication.en.md [hunk 4]
```diff
@@ -14,55 +14,55 @@ Receiving codes either from **SMS** or **email** are one of the weaker ways to s
...
 
 Unlike [FIDO2 / U2F](#fido2-u2f), TOTP offers no protection against [phishing](https://en.wikipedia.org/wiki/Phishing) or reuse attacks. If an adversary obtains a valid code from you they may use it as many times as they like until it expires (generally 60 seconds).
 
 An adversary could set up a website to imitate an official service in an attempt to trick you into giving out your username, password and current TOTP code. If the adversary then uses those recorded credentials they may be able to log into the real service and hijack the account.
 
-Although not perfect it is secure enough for most people, and when [Hardware Security Keys](#hardware-security-keys) are not supported TOTP with [Authenticator Apps](#authenticator-apps) are still a good option.
+Although not perfect it is secure enough for most people, and when [Hardware Security Keys](#hardware-security-keys) are not supported TOTP, with [authenticator apps](/multi-factor-authentication/#authenticator-apps) are still a good option.
```
Comment (jonaharagon) [inline docs/security/multi-factor-authentication.en.md:35]:
```suggestion
Although not perfect, TOTP is secure enough for most people, and when [Hardware Security Keys](#hardware-security-keys) are not supported [authenticator apps](/multi-factor-authentication/#authenticator-apps) are still a good option.
```

File: docs/security/multi-factor-authentication.en.md [hunk 5]
```diff
@@ -76,41 +76,41 @@ This presentation discusses the history of password authentication, the pitfalls
...
+### Which method should I use?
 
 When configuring your MFA method, keep in mind that it is only as secure as your weakest authentication method you use. This means it is important that you only use the best MFA method available. For instance, if you are already using TOTP, you should disable email and SMS MFA. If you are already using FIDO2 / U2F, you should not be using Yubico OTP or TOTP on your account.
 
 ### Backups
 
 You should always have backups for your MFA method. Hardware security keys can get lost, stolen, or simply stop working over time. It is recommended that you have a pair of hardware security keys with the same access to your accounts instead of just one.
 
-When using TOTP with an authenticator app, be sure to back up your recovery keys, the app itself, or copy the "shared secrets" to another instance of the app on a different phone or into an encrypted container (e.g. [VeraCrypt](/encryption/#veracrypt)).
+When using TOTP with an authenticator app, be sure to back up: your recovery keys, the app itself, or copy the "shared secrets" to another instance of the app on a different phone or into an encrypted container (e.g. [VeraCrypt](/encryption/#veracrypt)).
```
Comment (jonaharagon) [inline docs/security/multi-factor-authentication.en.md:97]:
```suggestion
When using TOTP with an authenticator app, be sure to back up your recovery keys or the app itself, or copy the "shared secrets" to another instance of the app on a different phone or to an encrypted container (e.g. [VeraCrypt](/encryption/#veracrypt)).
```

File: docs/security/multi-factor-authentication.en.md [hunk 6]
```diff
@@ -76,41 +76,41 @@ This presentation discusses the history of password authentication, the pitfalls
...
 
 ### Initial setup
 
-When buying a security key, it is important that you change the default credentials, setup password protection for the key, and enable touch confirmation if your key supports such feature. Products such as the [YubiKey](#yubikey) have multiple interfaces with seperate credentials for each one of them, so you should go over each interface and set up protection as well.
+When buying a security key, it is important that you change the default credentials, setup password protection for the key, and enable touch confirmation if your key supports such a feature. Products such as the [YubiKey](#yubico-otp) have multiple interfaces with seperate credentials for each one of them, so you should go over each interface and set up protection as well.
```
Comment (jonaharagon) [inline docs/security/multi-factor-authentication.en.md:101]:
```suggestion
When buying a security key, it is important that you change the default credentials, setup password protection for the key, and enable touch confirmation if your key supports it. Products such as the [YubiKey](#yubico-otp) have multiple interfaces with seperate credentials for each one of them, so you should go over each interface and set up protection as well.
```

**Review by jonaharagon — CHANGES_REQUESTED** [review]

File: docs/security/multi-factor-authentication.en.md (hunk 2 above)
Comment (0rdinant) [reply docs/security/multi-factor-authentication.en.md:7]: Maybe if it was written as "The idea behind MFA is that even if a hacker (or adversary) is able to figure out your password (something you *know*), they will still need a device you own —like your phone—(something you *have*) in order to generate the code needed to log in to your account."? I just don't like how the em dash is followed by the bracket, however if you don't add a pause in that sentence it runs on without a chance to breathe, assuming you read what's inside the brackets.

Comment (jonaharagon) [reply docs/security/multi-factor-authentication.en.md:7]: I might just drop it entirely:

> "The idea behind MFA is that even if a hacker (or adversary) is able to figure out your password (something you know), they will still need a device you own in order to generate the code (something you have) needed to log in to your account."

Comment (jonaharagon) [reply docs/security/multi-factor-authentication.en.md:7]:
```suggestion
The idea behind MFA is that even if a hacker (or adversary) is able to figure out your password (something you *know*), they will still need a device you own to generate the code (something you *have*) needed to log in to your account. MFA methods vary in security based on this premise: The more difficult it is for an attacker to gain access to your MFA method, the better. Examples of MFA methods (from weakest to strongest) include [SMS codes](https://en.wikipedia.org/wiki/One-time_password#SMS), email codes, app push notifications, [Time-based One-time Passwords (TOTP)](https://en.wikipedia.org/wiki/Time-based_one-time_password), Yubico OTP, and [FIDO2](https://en.wikipedia.org/wiki/FIDO2_Project) / [U2F](https://en.wikipedia.org/wiki/Universal_2nd_Factor).
```

Comment (dngray) [reply docs/security/multi-factor-authentication.en.md:7]: Well I thought about rewording it. I think it's that "The idea behind..."part.


> Normally, if a hacker (or adversary) is able to figure out your password then they’d gain access to the account that password belongs to. An account with MFA forces the hacker to have both the password (something you *know*) and a device that you own (something you *have*), like a phone.

> MFA methods vary in security based on the premise that the more difficult it is for an attacker to gain access to your MFA method, the better. Examples of MFA methods (from weakest to strongest) include [SMS codes](https://en.wikipedia.org/wiki/One-time_password#SMS), email codes, app push notifications, [Time-based One-time Passwords (TOTP)](https://en.wikipedia.org/wiki/Time-based_one-time_password), Yubico OTP, and [FIDO2](https://en.wikipedia.org/wiki/FIDO2_Project) / [U2F](https://en.wikipedia.org/wiki/Universal_2nd_Factor).

**Comment by dngray** [comment]
Should probably take a look at https://github.com/privacyguides/privacyguides.org/issues/999 before we merge this, might as well do them in one PR.

File: docs/security/multi-factor-authentication.en.md [hunk 7]
```diff
@@ -74,43 +84,43 @@ This presentation discusses the history of password authentication, the pitfalls
...
+When configuring your MFA method, keep in mind that it is only as secure as your weakest authentication method you use. This means it is important that you only use the best MFA method available. For instance, if you are already using TOTP, you should disable email and SMS MFA. If you are already using FIDO2/WebAuthn, you should not be using Yubico OTP or TOTP on your account.
 
 ### Backups
 
 You should always have backups for your MFA method. Hardware security keys can get lost, stolen, or simply stop working over time. It is recommended that you have a pair of hardware security keys with the same access to your accounts instead of just one.
 
-When using TOTP with an authenticator app, be sure to back up your recovery keys, the app itself, or copy the "shared secrets" to another instance of the app on a different phone or into an encrypted container (e.g. [VeraCrypt](../encryption.md#veracrypt)).
+When using TOTP with an authenticator app, be sure to back up your recovery keys or the app itself, or copy the "shared secrets" to another instance of the app on a different phone or to an encrypted container (e.g. [VeraCrypt](../encryption.md/#veracrypt)).
```
Comment (dngray) [inline docs/security/multi-factor-authentication.en.md:107]: This doesn't seem to work seems to think it's `//security/[encryption.md/#veracrypt](http://encryption.md/#veracrypt)`

Comment (dngray) [reply docs/security/multi-factor-authentication.en.md]: I fixed this i was a dumb bear

**Review by dngray — APPROVED** [review]

**Pull request merged**